README.md

```diff
@@ -11,21 +11,20 @@ just ahm paseo || echo "Setup failed"
 
 # Just commands
 
-We envision interaction with ahm-dryrun mainly using a set of `just` commands. You can adjust AHM dry-run to your needs using `.env` config file.
-
-You can run AHM for Polkadot using `just ahm polkadot` or substitute the runtime with a custom one by updating submodules to point to your branches/commits. There are many things you can do with `just` but, at first, you should not need anything other than these two. In case you want to contribute directly to the codebase, see Code Contribution section towards the end of th page.
-
-## Updating submodules
-
-Make sure to run `git submodule update --recursive` after you pull from the repo to update the submodules to the commits tracked in the repo.
-
-You can run e.g. `cd runtimes && git checkout <commit_hash> && cd - && git add runtimes && git commit` to update the commit that the runtimes submodule points to.
+- `just` to see the list of commands
+- `just init` to initialize the repo
+- `just setup` to install dependencies
+- `just ahm [paseo|polkadot]` to run the Migration for a given runtime. No args prints the help menu.
+- `just zb [bite|spawn|perform-migration]` to run the Zombie-Bite commands. No args prints the help menu.
+- `just e2e-tests` to run the E2E tests
+- `just wah-e2e-tests` to run the Westend Asset Hub E2E tests
+<!-- TODO @donal: Monitoring here -->
```

Comment:
yea finishing this after a call, will need some upstream fixes AFAICT - naively running the setup commands in the monitor doesn't work from a fresh clone. need time to look into it